composer.json

```diff
@@ -4,20 +4,23 @@
     "type": "project",
     "license": "proprietary",
     "require": {
-        "php": "^7.3.8",
+        "php": "^7.4.0",
         "ext-ctype": "*",
         "ext-iconv": "*",
-        "rollbar/rollbar-php-symfony-bundle": "v3.1.2",
-        "studio24/frontend": "0.6.*",
+        "rollbar/rollbar-php-symfony-bundle": "dev-master",
```

Comment:
I couldn't do `composer install` with this version, we need to use `"rollbar/rollbar-php-symfony-bundle": "4.*",`